assets/js/lib/model/activityLog.js

```diff
@@ -277,6 +281,24 @@ export const ACTIVITY_TYPES_CONFIG = {
     resource: (_entry) => 'Profile',
     allowedTo: userManagement,
   },
+  [PERSONAL_ACCESS_TOKEN_CREATION]: {
+    label: 'Personal Access Token Created',
+    message: (_entry) => `Personal access token was created`,
+    resource: (_entry) => 'Profile',
+    allowedTo: userManagement,
```

Comment:
by the way, if we no not mark PATs operations as allowed to user management, we'd need to make sure no sensitive information about the token is in the metadata.

Having these gated this way is a good starting point, and it highlights possible improvements in the activity log system when it comes to own actions and sensitive information.

Reply:
Let's be coherent of what we do with other profile changes by now.